Trell/CliCommands/InitCommand.cs

```diff
@@ -3,105 +3,209 @@
-                """);
+        // TODO: validate user input
+        userDataRootDirectory ??= AnsiConsole.Prompt(
+            new TextPrompt<string>("Please provide a path for where to store user data")
+            .DefaultValue(DEFAULT_USER_DATA_ROOT_DIR)
+            .ShowDefaultValue()
+        );
+
+        if (!Directory.Exists(userDataRootDirectory)) {
+            Directory.CreateDirectory(userDataRootDirectory);
+            Log.Information("Created {dir}", userDataRootDirectory);
+        }
+
+        if (!configAlreadyExists || userDataRootDirectory != config.Storage.Path) {
+            config.Storage.Path = userDataRootDirectory;
+            using var trellCfgFs = File.Open(CONFIG_FILE, mode: FileMode.OpenOrCreate, access: FileAccess.ReadWrite);
+            using var sw = new StreamWriter(trellCfgFs);
+            if (!config.TryConvertToToml(out var configAsText)) {
+                throw new TrellException("Unable to convert config to TOML");
             }
-            Log.Information("Wrote {file}", trellCfgFs.Name);
+            await sw.WriteLineAsync(configAsText);
+
+            Log.Information(configAlreadyExists ? "Updated existing {file}" : "Wrote {file}", CONFIG_FILE);
+        }
+
+        // TODO: validate user input for usernames and worker names, and/or expand TrellPath's valid characters.
+        // We use TrellPath.TryParseRelative to navigate to some of these folders, so if the names given contain
+        // any characters like 'ø' it can fail TrellPath's validation for paths
+        var username = settings.Username ?? AnsiConsole.Prompt(
+            new TextPrompt<string>("Please provide a username")
+            .DefaultValue("new-user")
+            .ShowDefaultValue()
+        );
+
+        var workerName = settings.WorkerName ?? AnsiConsole.Prompt(
+            new TextPrompt<string>("Please provide a name for a new worker")
+            .DefaultValue("new-worker")
+            .ShowDefaultValue()
+        );
+
+        MakeDirectoriesForNewWorker(userDataRootDirectory, username, workerName);
+        Log.Information("Created directories for ( user: {u}, worker: {w} )", username, workerName);
+
+        // Users need, at bare minimum, a template worker.js to start from
+        var workerPath = Path.GetFullPath(WORKER_FILE, GetWorkerSrcPath(userDataRootDirectory, username, workerName));
+        if (File.Exists(workerPath)) {
+            Log.Information("Worker file already exists at worker directory, skipping.");
         } else {
-            Log.Information("{f} already exists. skipping.", "Trell.config");
+            using var workerJsFs = File.Open(workerPath, mode: FileMode.CreateNew, access: FileAccess.ReadWrite);
+            using var sw2 = new StreamWriter(workerJsFs);
+            await sw2.WriteLineAsync("""
+                async function scheduled(event, env, ctx) {
+                    // Add work here you would like to schedule
+                    return "scheduled completed"
```

Comment:
```suggestion
                    console.log("running scheduled");
```

And the others. The comments aren't quite right and we don't return data from workers in this way.

Reply:
What should the comments say instead?

Reply:
Resolved in commit 6ea32ce